app/models/role.rb

```diff
@@ -68,6 +68,7 @@ def by_login login, account:
   end
 
   def password= password
+    Util::Password.validate_complexity password
```

Comment:
Validations like this should be added as a validation: https://guides.rubyonrails.org/active_record_validations.html. Let's refactor and move the logic down to the model responsible for the password.

Reply:
Just clarifying here that this should be a `sequel` validation, rather than active record:
- http://sequel.jeremyevans.net/rdoc/files/doc/validations_rdoc.html
- http://guides.rubyonrails.org/active_record_validations.html